File: Application/ARQuiz/app/src/main/java/com/lepko/martin/arquiz/Entities/Answer.java
package com.lepko.martin.arquiz.Entities;

/**
 * Created by Martin on 6.3.2017.
 */

public class Answer {

    private int id;
    private String name;
    private boolean isCorrect;

    public Answer(int id, String name, boolean isCorrect) {
        this.id = id;
        this.name = name;
        this.isCorrect = isCorrect;
    }

    public String getName() {
        return name;
    }

    public int getId() {
        return id;
    }

    public boolean isCorrect() {
        return isCorrect;
    }
}

File: Application/ARQuizServer/entities/Answer.php
<?php
/**
 * Created by PhpStorm.
 * User: Martin
 * Date: 13.2.2017
 * Time: 21:24
 */

namespace entities;


class Answer
{
    public static $correct = 1;
    public static $incorrect = 0;

    private $id;
    private $name;
    private $questionId;
    private $isCorrect;

    function __construct($id, $name, $questionId, $isCorrect)
    {
        $this->id = $id;
        $this->name = $name;
        $this->questionId = $questionId;
        $this->isCorrect = $isCorrect;
    }

    public function getId()
    {
        return $this->id;
    }

    public function getName()
    {
        return $this->name;
    }

    public function getIsCorrect()
    {
        return $this->isCorrect;
    }

    public function getQuestionId()
    {
        return $this->questionId;
    }
}